fhir-persistence/src/main/java/com/ibm/fhir/persistence/helper/FHIRTransactionHelper.java

```diff
@@ -88,4 +89,14 @@ public void rollback() {
             }
         }
     }
+
+    /**
+     * Use this only if the transaction is not started and managed by us, e.g, by JavaBatch framework.
+     */
+    public void commit2() throws FHIRPersistenceException {
```

Comment:
I don't think this is the right approach.

Reply:
changed to use existing commit()